README.md

```diff
@@ -74,7 +74,7 @@ client.changeUserLeverage({leverage: 4, symbol: 'ETHUSD'})
   });
 ```
 
-See inverse [rest-client.js](./lib/rest-client.js) for further information.
+See inverse [rest-client.ts](../typescript/src/rest-client.ts) & [rest api docs](../typescript/doc/rest-client.md) for further information.
```

Comment:
Good catch! Although it should just be `(./src/rest-client.ts)` instead of the original ./lib/ path, without the typescript part since that branch name is temporary.

On the docs, I'm actually looking to deprecate those. Ideally the code is as self documenting as possible going forward, with fallback to the official bybit api docs.

Reply:
I understand. When you merge to master I will add some useful examples to the readme.

Reply:
Would be great! I was thinking to introduce a runnable samples folder showing different workflows, but that's for later.